app/jobs/subscription_confirm_job.rb

```diff
@@ -47,13 +47,10 @@ def confirm_order!(order)
 
     process_payment!(order)
     send_confirmation_email(order)
+    send_payment_authorization_emails(order)
   rescue StandardError => e
-    if order.errors.any?
-      send_failed_payment_email(order)
-    else
-      Bugsnag.notify(e, order: order)
-      send_failed_payment_email(order, e.message)
-    end
+    send_failed_payment_email(order, order.errors.any? ? nil : e.message)
+    Bugsnag.notify(e, order: order) unless order.errors.any?
```

Comment:
if we go this way, I think it's simpler to make the condition positive like `if order.errors.present?`. Didn't Rubocop tell you that? werid.

Reply:
Do you mean like this `Bugsnag.notify(e, order: order) if order.errors.blank?`?

Reply:
I originally refactored this to reduce the method's line count (since I'd added a single line before the rescue clause and pushed it over 10 lines). I think it's much easier to read and understand written out, however, so I think we should just override the 10 line warning in rubocop instead and keep the logic the way it was.